app/connectors/exchange/ExportsDeclarationExchange.scala

```diff
@@ -234,8 +234,8 @@ object ExportsDeclarationExchange {
     )
     JsObject(values.flatten)
   }
-
-  implicit val format: OFormat[ExportsDeclarationExchange] = Json.format[ExportsDeclarationExchange]
```

Comment:
What about some tests for reads and writes? We should cover it in cases when we're making custom reads and writes